assets/src/edit-story/app/highlights/quickActions/useQuickActions.js

```diff
@@ -51,32 +66,119 @@ export const ACTION_TEXT = {
  * @return {Array.<MenuItemProps>} an array of quick action objects
  */
 const useQuickActions = () => {
-  const { currentPage, selectedElements } = useStory(
-    ({ state: { currentPage, selectedElements } }) => ({
+  const {
+    currentPage,
+    selectedElementAnimations,
+    selectedElements,
+    updateElementsById,
+  } = useStory(
+    ({
+      state: { currentPage, selectedElementAnimations, selectedElements },
+      actions: { updateElementsById },
+    }) => ({
       currentPage,
+      selectedElementAnimations,
       selectedElements,
+      updateElementsById,
     })
   );
-
+  const { undo } = useHistory(({ actions: { undo } }) => ({
+    undo,
+  }));
+  const { showSnackbar } = useSnackbar();
   const { setHighlights } = useHighlights(({ setHighlights }) => ({
     setHighlights,
   }));
 
+  /**
+   * Prevent quick actions menu from removing focus from the canvas.
+   */
+  const handleMouseDown = useCallback((ev) => {
+    ev.stopPropagation();
+  }, []);
+
+  /**
+   * Reset properties on an element. Shows a snackbar once the properties
+   * have been reset.
+   *
+   * @param {string} elementId the id of the element
+   * @param {Array.<string>} properties The properties of the element to update
+   * @return {void}
+   */
+  const handleResetProperties = useCallback(
+    (elementId, properties) => {
+      const newProperties = {};
+
+      // Choose properties to clear
+      if (properties.includes('backgroundOverlay')) {
+        newProperties.backgroundOverlay = null;
+      } else if (properties.includes('animation')) {
```

Comment:
I think this makes sense for this PR, but thinking about background elements (selfishly to reuse this since I'm working on that bit) - thoughts on updating this to be 2 separate if statements so that both could potentially get reset?